pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, trainer: 'pl.Trainer', log_gpu_memory: Optional[str] = None)
 
     def on_trainer_init(
         self,
-        logger: LightningLoggerBase,
+        logger: Union[bool, Iterable, LightningLoggerBase],
```

Comment:
Is it possible to be more precise with the `Iterable` type here? Would `Iterable[LightningLoggerBase]` work?

Reply:
@kandluis updated